client/consensus/babe/src/tests.rs

```diff
@@ -45,7 +45,7 @@ use sp_keystore::{
 	SyncCryptoStore,
 };
 use sp_runtime::{
-	generic::{Digest, DigestItem},
+	generic::{BlockId, Digest, DigestItem},
```

Comment:
Why this import?

Reply:
> Why this import?

This import was removed from [`lib.rs`](https://github.com/paritytech/substrate/blob/a809327b0c7ace4e8757f2e65ed7b88a7c75b198/client/consensus/babe/src/lib.rs#L126). And this `mod tests` is still using  `BlockId`.